dev-util/webstorm/webstorm-2017.2.2.172.3757.55.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils versionator
+
+SLOT="$(get_major_version)"
+MY_PV="$(get_version_component_range 4-6)"
+MY_PN="WebStorm"
+
+DESCRIPTION="JavaScript IDE for client- and server-side development with Node.js"
+HOMEPAGE="http://www.jetbrains.com/webstorm"
+SRC_URI="http://download.jetbrains.com/${PN}/${MY_PN}-$(get_version_component_range 1-3).tar.gz"
+
+LICENSE="WebStorm WebStorm_Academic WebStorm_Classroom WebStorm_OpenSource WebStorm_personal"
+IUSE=""
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=virtual/jdk-1.7"
+S="${WORKDIR}/${MY_PN}-${MY_PV}"
+
+src_install() {
+	local dir="/opt/${PN}"
```

Comment:
This variable hurts readability for little benefit. Kill it. Also, don't use `${PN}` here proactively.

Reply:
variable 'dir' removed.
What do you mean with 'proactively'?

Reply:
By 'proactively' I mean when it's not really required.